src/opm/output/eclipse/AggregateGroupData.cpp

```diff
@@ -716,7 +716,7 @@ void staticContrib(const Opm::Schedule&     sched,
                         iGrp[nwgmax + 17] = 0;
                     }
                     else if (higher_lev_winj_ctrl > 0 || higher_lev_winj_cmode > 0) {
-                        if ((winj_cmode != Opm::Group::InjectionCMode::FLD) || (winj_cmode!= Opm::Group::InjectionCMode::NONE)) {
+                        if (!((winj_cmode == Opm::Group::InjectionCMode::FLD) || (winj_cmode == Opm::Group::InjectionCMode::NONE))) {
```

Comment:
This change **appears** to influence `IGRP[nwgmax + 17]` for the `CENTRAL` group of [MOD4_GRP](https://ci.opm-project.org/job/opm-common-PR-builder/2304/testReport/junit/(root)/mpi/compareECLFiles_flow_MOD4_GRP/) and [MOD4_UDQ_ACTIONX]() on report steps 38 and 51.  The value is reset from `3` to `1` in both test cases.

Would you be able to confirm/verify that this behaviour is expected?

Reply:
Yes - this change is OK/expected and gives the value it should have in this case!

Reply:
> Yes - this change is OK/expected and gives the value it should have in this case!

Excellent.  Thanks a lot for confirming the expected behaviour.